pom.xml

```diff
@@ -100,6 +101,7 @@
 
 	<properties>
 		<camel-spring-boot-version>4.8.3.redhat-00003</camel-spring-boot-version>
+		<maven-compiler-plugin-version>3.13.0</maven-compiler-plugin-version>
```

Comment:
Can we use the maven-compiler-plugin-version property from camel? https://github.com/jboss-fuse/camel/blob/camel-4.8.3-branch/pom.xml#L130

Reply:
nope, it cannot be inherited from there.